src/hooks/usePlayList.js

```diff
@@ -1,87 +1,64 @@
+// @ts-check
 import { useEffect, useState } from 'react'
 import getPlaylistData from '../utils/getPlaylistData'
+import getWrongAnswers from '../utils/getWrongAnswers'
 import shuffle from '../utils/shuffleArray'
 
 export default function usePlayList(playlist) {
-  const newPlaylist = shuffle(playlist)
+  const newPlaylist = playlist
   const [playlistData, setPlaylistData] = useState(null)
   const [answers, setAnswers] = useState(null)
   const [counter, setCounter] = useState(null)
   const [wrongAnswers, setWrongAnswers] = useState(null)
   let getNextUrl
 
-  // useEffect(()=> {
-
-  // })
-
-  //get playlist data
+  // get playlist data
```

Comment:
Why not just call the function `getPlaylistData` and remove the comment?

Reply:
This was due to uniformity.